src/utils.test.js

```diff
@@ -238,7 +238,7 @@ describe('utils', () => {
         invokedFunctionArn: 'arn:aws:lambda:region:account:function:name:alias',
       },
     });
-    utils.setValidAliases(['wrong']);
+    process.env['LUMIGO_VALID_ALIASES'] = '["wrong"]';
```

Comment:
It's a complicated way to write it, why not `wrong,right` and then split & trim it?

Reply:
I don't understand

Reply:
in our documentation, it's the way I wrote it